packages/panels/src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -17,33 +17,45 @@ import {
 export interface SidebarMenuProps extends BoxProps {
   onCloseClick?: SidebarMenuCloseButtonProps["onClick"];
   hideCloseButton?: boolean;
+  collapsed?: boolean;
+  variant?: SidebarMenuVariant;
 }
 
-// TODO Add variant
+export type SidebarMenuVariant = "light" | "dark";
```

Comment:
There might be other "dimensions" later. Like `"compact" | "standard"`.
It would not be possible to combine these with `dark` if they are all part of the variant type.

So I would change the `dark` to just be a CSS class that works as a theme.